src/classes/Gift.cls

```diff
@@ -68,14 +68,14 @@ public with sharing class Gift {
         return this.hasPaymentFieldsChanged();
     }
 
-    public DataImport__c dataImportRecord() {
-        return this.dataImport;
-    }
-
     public SoftCredits softCredits() {
         return this.softCredits;
     }
 
+    public Map<String, Object> fieldsWithValues() {
```

Comment:
What do you think about naming this 'fields' also for consistency?

Reply:
Agree!